packages/calcite-components/src/components/table/table.tsx

```diff
@@ -0,0 +1,524 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop,
+  State,
+  VNode,
+  Watch,
+} from "@stencil/core";
+import { Scale, SelectionMode } from "../interfaces";
+import {
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent,
+} from "../../utils/loadable";
+import {
+  connectMessages,
+  disconnectMessages,
+  setUpMessages,
+  T9nComponent,
+  updateMessages,
+} from "../../utils/t9n";
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  numberStringFormatter,
+  NumberingSystem,
+} from "../../utils/locale";
+import { TableLayout, TableRowFocusEvent } from "./interfaces";
+import { CSS, SLOTS } from "./resources";
+import { TableMessages } from "./assets/table/t9n";
+import { getUserAgentString } from "../../utils/browser";
+
+/**
+ * @slot - A slot for adding `calcite-table-row` or nested `calcite-table` elements. Content placed here will be rendered in a `table-body` tag.
```

Comment:
Does the `table-body` tag need to be mentioned, or if it should be mentioned, how does this impact the developer?

Reply:
Yeah, let's omit that.